doc/releases/release-notes-3.7.rst

```diff
@@ -427,6 +444,13 @@ Drivers and Sensors
 
 * I3C
 
+  * Added shell support for querying bus and CCC commands.
+
+  * Added driver to support the I3C controller on NPCX.
+
+  * Improvements and bug fixes on ``i3c_mcux``, including handling the bus being busy
```

Comment:
Here and elsewhere in your PR, it might be worth referencing drivers using their actual compatible and use `:dtcompatible:`

```suggestion
  * Improvements and bug fixes on :dtcompatible:`nxp,mcux-i3c`, including handling the bus being busy
```

Reply:
I have added the dtcompatibles. I want to keep the driver file names so it is easier to cross reference.